src/VirtoCommerce.PriceExportImportModule.Core/ModuleConstants.cs

```diff
@@ -59,26 +59,26 @@ public static class General
             {
                 public static SettingDescriptor ExportLimitOfLines { get; } = new SettingDescriptor
                 {
-                    Name = "SimpleExportImport.Export.LimitOfLines",
-                    GroupName = "SimpleExportImport|Export",
+                    Name = "PriceExportImport.Export.LimitOfLines",
```

Comment:
Don't forget to change https://github.com/VirtoCommerce/vc-webstore-deploy/blob/qa/webstore-app/resources/platform-cm.yaml#L29-L31 or add info to QA issue

Reply:
Done https://virtocommerce.atlassian.net/browse/VDS-1701